README.md

```diff
@@ -30,292 +31,4 @@ var constructorio = new ConstructorIOClient({
-
-#### Send conversion event
-```javascript
-constructorio.tracker.trackConversion('dogs', {
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `name` | string | ✔ |
-| `customer_id` | string | ✔ |
-| `variation_id` | string | ✔ |
-| `revenue` | string | ✔ |
-| `section` | string | |
-| `result_id` | string | |
-
-#### Send purchase event
-```javascript
-constructorio.tracker.trackPurchase({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `items` | array | ✔ |
-| `revenue` | string | ✔ |
-| `order_id` | string | |
-| `section` | string | |
-
-#### Send recommendation view event
-```javascript
-constructorio.tracker.trackRecommendationView({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `url` | string | ✔ |
-| `pod_id` | string | ✔ |
-| `num_results_viewed` | number | ✔ |
-| `result_count` | integer | |
-| `result_page` | integer | |
-| `result_id` | string | |
-| `section` | string | |
-
-#### Send recommendation click event
-```javascript
-constructorio.tracker.trackRecommendationClick({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `pod_id` | string | ✔ |
-| `strategy_id` | string | ✔ |
-| `item_id` | string | ✔ |
-| `result_count` | integer | |
-| `result_page` | integer | |
-| `result_id` | string | |
-| `section` | string | |
-| `variation_id` | string | |
-| `num_results_per_page` | integer | |
-| `result_position_on_page` | integer | |
-
-#### Send browse results load event
-```javascript
-constructorio.tracker.trackBrowseResultsLoaded({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `url` | string | ✔ |
-| `filter_name` | string | ✔ |
-| `filter_value` | string | ✔ |
-| `section` | string | |
-| `result_id` | string | |
-| `result_count` | integer | |
-| `result_page` | integer | |
-| `selected_filters` | string or array | |
-| `sort_by` | string | |
-| `sort_order` | string | |
-| `items` | array | |
-
-#### Send browse result click event
-```javascript
-constructorio.tracker.trackBrowseResultClick({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `item_id` | string | ✔ |
-| `filter_name` | string | ✔ |
-| `filter_value` | string | ✔ |
-| `variation_id` | string | |
-| `section` | string | |
-| `result_id` | string | |
-| `result_count` | integer | |
-| `result_page` | integer | |
-| `result_position_on_page` | integer | |
-| `num_results_per_page` | integer | |
-| `selected_filters` | string or array | |
-
-#### Receive status of tracking requests
-The status of tracking requests can be observed through emitted events. `messageType` must be either "success" or "error", and the `callback` parameter must be a function that will be passed the message data as the first and only argument.
-```javascript
-constructorio.tracker.on(messageType, callback);
-```
-
-#### Dispatched events
```

Comment:
Same with this

Reply:
This one we might want to leave in the readme - not sure how we'd include this concept in the JSDocs as it's a bit outside of core code.

Reply:
Cool, yea. Sounds good to me.